### PR TITLE
Implemented Capacity Dashboard in Track

### PR DESCRIPTION
**Pull request by srikanthv02**
@hrushikesh07 : Please verify and merge.

### CODE REVIEW

File: client/htmls/public/css/capacity.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.center-block {
+     height: 50px; 
+     width: 50px; 
+     margin-top: 20%; 
+     margin-left: 48%;
+     margin-bottom: 10%; 
+     position: absolute;
+}
+
+tr, tr input {
```
Comment (hrushikesh07) [inline client/htmls/public/css/capacity.css]: use parent selector .

File: client/htmls/public/css/capacity.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ .marginTop20 {
+     margin-top: 20px;
+ }
+
+ .marginBottom15 {
+     margin-bottom: 15px;
+     font-size: 16px;
+ }
+
+ .footer {
```
Comment (hrushikesh07) [inline client/htmls/public/css/capacity.css]: use parent selector

File: client/htmls/public/js/awsCapacity.js [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+            urlManagedNoProvider = "../tracked-instances?category=managed&filterBy=orgId:" + orgId + ' ' + "providerId:" + orgProviderId;
+        } else {
+            urlManagedProvider = "../tracked-instances?category=managed&filterBy=orgId:" + orgId;
+        }
+        $.ajax({
+            type: "get",
+            dataType: "json",
+            async: false,
+            url: (urlManagedNoProvider) ? urlManagedNoProvider : urlManagedProvider,
+            success: function(data) {
```
Comment (hrushikesh07) [inline client/htmls/public/js/awsCapacity.js]: error part is missing for error handling.

File: client/htmls/public/js/awsCapacity.js [hunk 4]
```diff
@@ -0,0 +1,437 @@
...
+            urlManagedNoProvider = "../tracked-instances?category=assigned&filterBy=orgId:" + orgId + ' ' + "providerId:" + orgProviderId;
+        } else {
+            urlManagedProvider = "../tracked-instances?category=assigned&filterBy=orgId:" + orgId;
+        }
+        $.ajax({
+            type: "get",
+            dataType: "json",
+            async: false,
+            url: (urlManagedNoProvider) ? urlManagedNoProvider : urlManagedProvider,
+            success: function(data) {
```
Comment (hrushikesh07) [inline client/htmls/public/js/awsCapacity.js]: error part is missing for error handling.

File: client/htmls/public/js/awsCapacity.js [hunk 5]
```diff
@@ -0,0 +1,437 @@
...
+            urlManagedNoProvider = "../tracked-instances?category=unassigned&filterBy=orgId:" + orgId + ' ' + "providerId:" + orgProviderId;
+        } else {
+            urlManagedProvider = "../tracked-instances?category=unassigned&filterBy=orgId:" + orgId;
+        }
+        $.ajax({
+            type: "get",
+            dataType: "json",
+            async: false,
+            url: (urlManagedNoProvider) ? urlManagedNoProvider : urlManagedProvider,
+            success: function(data) {
```
Comment (hrushikesh07) [inline client/htmls/public/js/awsCapacity.js]: error part is missing for error handling.

**Comment by srikanthv02** [comment]
@hrushikesh07 : PR comments have been taken care. Please check

**Pull request merged**